uplink/storage/segments/store.go

```diff
@@ -151,19 +151,11 @@ func (s *segmentStore) Put(ctx context.Context, streamID storj.StreamID, data io
 	return Meta{}, nil
 }
 
-// Get requests the satellite to read a segment and downloaded the pieces from the storage nodes
-func (s *segmentStore) Get(ctx context.Context, streamID storj.StreamID, segmentIndex int32, objectRS storj.RedundancyScheme) (rr ranger.Ranger, _ storj.SegmentEncryption, err error) {
-	defer mon.Task()(&ctx)(&err)
-
-	info, limits, err := s.metainfo.DownloadSegment(ctx, metainfo.DownloadSegmentParams{
-		StreamID: streamID,
-		Position: storj.SegmentPosition{
-			Index: segmentIndex,
-		},
-	})
-	if err != nil {
-		return nil, storj.SegmentEncryption{}, Error.Wrap(err)
-	}
+// Get requests the satellite to read a segment and downloaded the pieces from the storage nodes.
```

Comment:
we need to adjust comment because now there is no request to satellite anymore here.

Reply:
I've just renamed the method and rewritten this comment.